.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/FineTuningTests.cs

```diff
@@ -91,17 +91,24 @@ public async Task EventsFineTuning()
     {
         string fineTunedModel = GetFineTunedModel();
         FineTuningClient client = GetTestClient();
-
         // Check if the model exists by searching all jobs
         FineTuningJob job = await EnumerateJobsAsync(client)
             .FirstOrDefaultAsync(j => j.FineTunedModel == fineTunedModel)!;
+
         Assert.That(job, Is.Not.Null);
         Assert.That(job!.Status, Is.EqualTo("succeeded"));
 
         HashSet<string> ids = new();
 
         int count = 25;
-        var asyncEnum = EnumerateAsync<FineTuningJobEvent>((after, limit, opt) => client.GetJobEventsAsync(job.ID, after, limit, opt));
+        var asyncEnum = EnumerateAsync<FineTuningJobEvent>((after, limit, opt) =>
+        {
+            var jobIdBinaryString = $"{{\"jobId\":\"{job.ID}\"}}";
+            var createdToken = ContinuationToken.FromBytes(BinaryData.FromString(jobIdBinaryString));
```

Comment:
Hm, I probably wouldn't put this in a test suite in a public repo.  ContinuationTokens are intended to be opaque, and I think it could be misleading to any resourceful developer looking for usage samples in a test base if they see an example constructing one directly.  If you need this for reasons other than the ones we've discussed offline, let's talk.  Ideally we address any real customer needs via public APIs that don't involve calling `ContinuationToken.FromBytes` with anything other than a value persisted from a call to `ContinuationToken.ToBytes`.  If you feel like this is unclear from the [public refdocs](https://learn.microsoft.com/en-us/dotnet/api/system.clientmodel.continuationtoken) for the type, let's file an issue to update those, so we can help others avoid trying to do this.

Reply:
Per offline discussion, we can address this in a follow-up PR if needed.  I would like to resolve it before merging the full change into main.